packages/emotion/src/useTheme.ts

```diff
@@ -37,7 +37,7 @@ import { isEmpty } from '@instructure/ui-utils'
 export const useTheme = () => {
   let theme = useEmotionTheme()
 
-  if (isEmpty(theme)) {
+  if (isEmpty(theme as Record<string, unknown>)) {
```

Comment:
I would move this `as Record<string, unknown>` type casting to be the return type of `useEmotionTheme` and then it would not be required us to cast it here.
What do you think?

Reply:
useEmotionTheme returns a `Theme` type, which comes from Emotion, which is an empty interface. Since this PR is about `ui-utils` I wanted to make minimal changes to the `emotion` package.